blocks/location-overview/location-overview.js

```diff
@@ -0,0 +1,94 @@
+import {
+  domEl,
+  div,
+  a,
+  span,
+  h2,
+  p,
+} from '../../scripts/dom-helpers.js';
+
+import {
+  decorateIcons,
+} from '../../scripts/aem.js';
+
+function getLearnMoreIcon() {
+  const learnMoreIcon = domEl('calcite-icon', {
+    icon: 'arrowRight',
+    scale: 's',
+  });
+
+  const iconContainer = div({ class: 'learn-more-icon-container' }, 'Learn more', learnMoreIcon);
+  return iconContainer;
+}
+
+function getPanelIcon(iconPath = '') {
+  const panelIcon = span({
+    class: iconPath || 'location-overview-panel-icon icon icon-2-5d-mesh-48',
```

Comment:
The icon should only be defined in the content, is there a reason to provide a fallback?

Reply:
In general is there a reason to create a span? The span is already generated when the icon is included in the content anyway

Reply:
> is there a reason to provide a fallback?

Nope, its there only while we import those icons, otherwise it would be an empty container.

> In general is there a reason to create a span?

Yes, that span doesn't have a class included. I can modify the function to accept a class and add it that is cleaner.

Reply:
Why do you need that class in the first place? If you want to reference that span from css you can just use ".icon"

Reply:
Rather than use "icon-2-5d-mesh-48" you could always add the icon directly in the content as `:2-5d-mesh-48:`, that should have the same effect and allow you to test the exact behaviour.

Reply:
mmm. Sure I can add it directly. I thought we use "decorateIcons" to have a consistent import strategy, hence the string with the prefix: icon-2-5d-mesh-48

Reply:
My point is that this icon should be always set in the content, not via code, can you add it in the content and ensure it works as intended?